src/ui/buttons/buttonDesigns/index.js

```diff
@@ -31,6 +41,18 @@ const DESIGN_MAP : Object = {
         getValidDesignProps:   getValidInlineLogoTextProps,
         designConfig:          INLINE_LOGO_TEXT_CONFIG,
         ButtonDesignComponent: InlineLogoTextComponent
+    },
+    'alternate-slide-logo-animation': {
+        designFn:              getAlternateSlideLogoAnimation,
+        getValidDesignProps:   getAlternateSlideLogoProps,
+        designConfig:          ALTERNATE_SLIDE_LOGO_CONFIG,
+        ButtonDesignComponent: AlternateSlideLogoComponent
+    },
+    'hover-slide-logo-animation': {
+        designFn:              getHoverSlideLogoAnimation,
+        getValidDesignProps:   getHoverSlideLogoProps,
+        designConfig:          ALTERNATE_SLIDE_LOGO_CONFIG,
```

Comment:
Can you add a comment to explain why we are naming this ALTERNATE_SLIDE for the hover animation or name it something that fits both?

Reply:
yeah is because I am reusing the same config, but I am going to create a general config structure to re use it wherever we can.